addons/website/static/src/scss/website.scss

```diff
@@ -1060,6 +1060,26 @@ table.table_desc tr td {
         }
     }
 }
+// Figure with special style
+.figure.o_website_relative_layout {
+    position: relative;
+    .figure-img {
+        width: 100%;
+        margin-bottom: 0;
+    }
+    figcaption {
+        @include o-position-absolute($left: 0, $bottom: 0);
+        background-color: rgba(0, 0, 0, 0.4);
+        opacity: 0.85;
+        width: 100%;
```

Comment:
You could avoid that rule by putting `$right: 0` on the `o-position-absolute` rule.